scopes/workspace/install/install.main.runtime.ts

```diff
@@ -199,14 +218,17 @@ export class InstallMain {
       }
       return component;
     });
-    if (!nonLoadedEnvs.length) {
+    if (nonLoadedEnvs.length) {
+      runInstall = true;
+    }
+    if (!runInstall) {
       this.workspace.inInstallContext = false;
       return;
     }
-    this.logger.consoleWarning(
-      `the following environments are not installed yet: ${nonLoadedEnvs.join(', ')}. installing them now...`
-    );
-    await this.install();
+    // this.logger.console(
+    // `the following environments are not installed yet: ${nonLoadedEnvs.join(', ')}. installing them now...`
+    // );
```

Comment:
You can still print this warning in this scenario, if you want. Why keeping it in comments?

Reply:
I was considering it, commented and added it back, and commented again.
Just not sure if this is really interesting info for the user. maybe as a regular log but not a warning.